src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -194,42 +201,45 @@ protected JsonNode getJSONFromFile(String filePath) throws IOException {
     /**
      * Wait for certain amount of events to be processed.
      *
-     * @param eventsCount - An int which indicated how many events that should be
-     *                    processed.
+     * @param eventsCount - An int which indicated how many events that should be processed.
      * @return
      * @throws InterruptedException
      */
     protected void waitForEventsToBeProcessed(int eventsCount) throws InterruptedException {
         // wait for all events to be processed
-        long stopTime = System.currentTimeMillis() + 60000;
+        long stopTime = System.currentTimeMillis() + SECONDS_30;
         long processedEvents = 0;
         while (processedEvents < eventsCount && stopTime > System.currentTimeMillis()) {
             processedEvents = countProcessedEvents(database, event_map);
             LOGGER.debug("Have gotten: " + processedEvents + " out of: " + eventsCount);
-            TimeUnit.MILLISECONDS.sleep(1000);
+            TimeUnit.MILLISECONDS.sleep(SECONDS_1);
+        }
+
+        if (processedEvents < eventsCount) {
+            fail(String.format(
+                    "IE did not process all sent events. Processed '%s' events out of '%s' sent.",
```

Comment:
It should be Eiffel Intelligence right, not Internet Explorer 😄

Reply:
Done